x-pack/plugin/inference/src/main/java/org/elasticsearch/xpack/inference/services/jinaai/embeddings/JinaAIEmbeddingsServiceSettings.java

```diff
@@ -144,18 +181,16 @@ public String getWriteableName() {
     public XContentBuilder toXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject();
 
-        builder = commonSettings.toXContentFragment(builder, params);
+        toXContentFragmentOfExposedFields(builder, params);
+
         if (similarity != null) {
```

Comment:
How about we move similarity and maxInputTokens to `toXContentFragmentOfExposedFields`